R/blpAuthenticate.R

```diff
@@ -67,17 +71,20 @@ blpAuthenticate <- function(uuid=getOption("blpUUID", NULL),
                             ip.address=getOption("blpLoginIP", NULL),
                             con=defaultConnection(),
                             default=TRUE,
-                            isEmrsId = getOption("isEmrsId", FALSE)) {
+                            isAuthId = getOption("blpIsAuthId", FALSE),
+                            appName = getOption("blpAppName", NULL)) {
     if(is.null(uuid)) {
         ## no UUID, assume B-PIPE or SAPI with application ID
         blpAuth <- authenticate_Impl(con, NULL, NULL, FALSE)
-    } else if(isEmrsId) {
-        if(is.null(uuid) || is.null(ip.address)){
-            stop("when isEmrsId is TRUE, uuid and ip.address must be passed")
+    } else if(isAuthId) {
```

Comment:
The existing else clause has some logic to resolve IP from host that I think would be useful here too. I think this chunk should get bumped down to right before the `authenticate_Impl` call.

Reply:
thanks - just took a stab at addressing this